pkgs/tools/admin/upcloud-cli/default.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "upcloud-cli";
+  version = "3.13.0";
+
+  src = fetchFromGitHub {
+    owner = "UpCloudLtd";
+    repo = "upcloud-cli";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-0hXqqLqZLX9zkRk6/sOh3/w+v5piuAV1277Nk6w1i6s=";
+  };
+
+  vendorHash = "sha256-Sl9D8PyqWfTBrPsVxJtLmU3Qxe5f3kpwd5Oi+5YZ4oo=";
+
+  doCheck = false;
```

Comment:
Please enable tests and only disable the ones that reach Internet

Reply:
Should I exhaustively try to find tests that attempt to reach the internet, or is it fine that I was able to run all the tests successfully with no connection?

Reply:
okie dokie, well none fail ^_^

<img width="772" alt="Screenshot 2024-12-25 at 23 13 07" src="https://github.com/user-attachments/assets/2615bb6c-c6a0-449b-9059-2b96708e2db6" />